affected_purls:
- pkg:pypi/tensorflow@0.12.0
- pkg:pypi/tensorflow@0.12.1
- pkg:pypi/tensorflow@1.0.0
- pkg:pypi/tensorflow@1.0.1
- pkg:pypi/tensorflow@1.1.0
- pkg:pypi/tensorflow@1.10.0
- pkg:pypi/tensorflow@1.10.1
- pkg:pypi/tensorflow@1.11.0
- pkg:pypi/tensorflow@1.12.0
- pkg:pypi/tensorflow@1.12.2
- pkg:pypi/tensorflow@1.12.3
- pkg:pypi/tensorflow@1.13.1
- pkg:pypi/tensorflow@1.13.2
- pkg:pypi/tensorflow@1.14.0
- pkg:pypi/tensorflow@1.15.0
- pkg:pypi/tensorflow@1.15.2
- pkg:pypi/tensorflow@1.15.3
- pkg:pypi/tensorflow@1.15.4
- pkg:pypi/tensorflow@1.15.5
- pkg:pypi/tensorflow@1.2.0
- pkg:pypi/tensorflow@1.2.1
- pkg:pypi/tensorflow@1.3.0
- pkg:pypi/tensorflow@1.4.0
- pkg:pypi/tensorflow@1.4.1
- pkg:pypi/tensorflow@1.5.0
- pkg:pypi/tensorflow@1.5.1
- pkg:pypi/tensorflow@1.6.0
- pkg:pypi/tensorflow@1.7.0
- pkg:pypi/tensorflow@1.7.1
- pkg:pypi/tensorflow@1.8.0
- pkg:pypi/tensorflow@1.9.0
- pkg:pypi/tensorflow@2.0.0
- pkg:pypi/tensorflow@2.0.1
- pkg:pypi/tensorflow@2.0.2
- pkg:pypi/tensorflow@2.0.3
- pkg:pypi/tensorflow@2.0.4
- pkg:pypi/tensorflow@2.1.0
- pkg:pypi/tensorflow@2.1.1
- pkg:pypi/tensorflow@2.1.2
- pkg:pypi/tensorflow@2.1.3
- pkg:pypi/tensorflow@2.1.4
- pkg:pypi/tensorflow@2.2.0
- pkg:pypi/tensorflow@2.2.1
- pkg:pypi/tensorflow@2.2.2
- pkg:pypi/tensorflow@2.2.3
- pkg:pypi/tensorflow@2.3.0
- pkg:pypi/tensorflow@2.3.1
- pkg:pypi/tensorflow@2.3.2
- pkg:pypi/tensorflow@2.3.3
- pkg:pypi/tensorflow@2.4.0
- pkg:pypi/tensorflow@2.4.1
- pkg:pypi/tensorflow@2.4.2
- pkg:pypi/tensorflow@2.5.0
- pkg:pypi/tensorflow-cpu@1.15.0
- pkg:pypi/tensorflow-cpu@2.1.0
- pkg:pypi/tensorflow-cpu@2.1.1
- pkg:pypi/tensorflow-cpu@2.1.2
- pkg:pypi/tensorflow-cpu@2.1.3
- pkg:pypi/tensorflow-cpu@2.1.4
- pkg:pypi/tensorflow-cpu@2.2.0
- pkg:pypi/tensorflow-cpu@2.2.1
- pkg:pypi/tensorflow-cpu@2.2.2
- pkg:pypi/tensorflow-cpu@2.2.3
- pkg:pypi/tensorflow-cpu@2.3.0
- pkg:pypi/tensorflow-cpu@2.3.1
- pkg:pypi/tensorflow-cpu@2.3.2
- pkg:pypi/tensorflow-cpu@2.3.3
- pkg:pypi/tensorflow-cpu@2.4.0
- pkg:pypi/tensorflow-cpu@2.4.1
- pkg:pypi/tensorflow-cpu@2.4.2
- pkg:pypi/tensorflow-cpu@2.5.0
- pkg:pypi/tensorflow-gpu@0.12.0
- pkg:pypi/tensorflow-gpu@0.12.1
- pkg:pypi/tensorflow-gpu@1.0.0
- pkg:pypi/tensorflow-gpu@1.0.1
- pkg:pypi/tensorflow-gpu@1.1.0
- pkg:pypi/tensorflow-gpu@1.10.0
- pkg:pypi/tensorflow-gpu@1.10.1
- pkg:pypi/tensorflow-gpu@1.11.0
- pkg:pypi/tensorflow-gpu@1.12.0
- pkg:pypi/tensorflow-gpu@1.12.2
- pkg:pypi/tensorflow-gpu@1.12.3
- pkg:pypi/tensorflow-gpu@1.13.1
- pkg:pypi/tensorflow-gpu@1.13.2
- pkg:pypi/tensorflow-gpu@1.14.0
- pkg:pypi/tensorflow-gpu@1.15.0
- pkg:pypi/tensorflow-gpu@1.15.2
- pkg:pypi/tensorflow-gpu@1.15.3
- pkg:pypi/tensorflow-gpu@1.15.4
- pkg:pypi/tensorflow-gpu@1.15.5
- pkg:pypi/tensorflow-gpu@1.2.0
- pkg:pypi/tensorflow-gpu@1.2.1
- pkg:pypi/tensorflow-gpu@1.3.0
- pkg:pypi/tensorflow-gpu@1.4.0
- pkg:pypi/tensorflow-gpu@1.4.1
- pkg:pypi/tensorflow-gpu@1.5.0
- pkg:pypi/tensorflow-gpu@1.5.1
- pkg:pypi/tensorflow-gpu@1.6.0
- pkg:pypi/tensorflow-gpu@1.7.0
- pkg:pypi/tensorflow-gpu@1.7.1
- pkg:pypi/tensorflow-gpu@1.8.0
- pkg:pypi/tensorflow-gpu@1.9.0
- pkg:pypi/tensorflow-gpu@2.0.0
- pkg:pypi/tensorflow-gpu@2.0.1
- pkg:pypi/tensorflow-gpu@2.0.2
- pkg:pypi/tensorflow-gpu@2.0.3
- pkg:pypi/tensorflow-gpu@2.0.4
- pkg:pypi/tensorflow-gpu@2.1.0
- pkg:pypi/tensorflow-gpu@2.1.1
- pkg:pypi/tensorflow-gpu@2.1.2
- pkg:pypi/tensorflow-gpu@2.1.3
- pkg:pypi/tensorflow-gpu@2.1.4
- pkg:pypi/tensorflow-gpu@2.2.0
- pkg:pypi/tensorflow-gpu@2.2.1
- pkg:pypi/tensorflow-gpu@2.2.2
- pkg:pypi/tensorflow-gpu@2.2.3
- pkg:pypi/tensorflow-gpu@2.3.0
- pkg:pypi/tensorflow-gpu@2.3.1
- pkg:pypi/tensorflow-gpu@2.3.2
- pkg:pypi/tensorflow-gpu@2.3.3
- pkg:pypi/tensorflow-gpu@2.4.0
- pkg:pypi/tensorflow-gpu@2.4.1
- pkg:pypi/tensorflow-gpu@2.4.2
- pkg:pypi/tensorflow-gpu@2.5.0
aliases:
- CVE-2021-35958
fixed_purl:
- pkg:pypi/tensorflow@2.6.0rc0
references:
- id: 183616
  reference_id: CVE-2021-35958
  url: https://nvd.nist.gov/vuln/detail/CVE-2021-35958
severities: []
summary: 'Exposure of Resource to Wrong Sphere

  TensorFlow allows attackers to overwrite arbitrary files via a crafted archive when
  `tf.keras.utils.get_file` is used with `extract=True`. NOTE: the vendor''s position
  is that `tf.keras.utils.get_file` is not intended for untrusted archives.'
vulnerability_id: VCID-zfe2-j7qc-aaaj
weaknesses: []
